### ISSUE TITLE
Vuejs Integration

### ISSUE THREAD

**Issue by andsilmar** [softarc-consulting/sheriff#159]
First of all, I want to thank you for this useful library. I’ve been using it in an Angular project without any issues.

Now, I would like to integrate the library into a Vue.js project, but I'm having some trouble getting it to work properly. Is there any special configuration needed for *.vue files that I might be overlooking?

Thanks!

**Comment by rainerhahnekamp** [comment]
@andsilmar, thanks; I've been waiting for that question for some time. We will require an adapter for Vue integration that can also parse those files.

Would you be interested in acting as "testing candidate"?

**Comment by andsilmar** [comment]
Yes, of course! I would be happy to help. Feel free to send the instructions when you have them.

**Comment by rainerhahnekamp** [comment]
@andsilmar Here's an update on the Vue support. A colleague has managed to use Sheriff in a Vue project. According to him, it just worked without any issues. I still have to try it out but if the existing TypeScript configuration can parse vue files, it should just work.

**Comment by andsilmar** [comment]
Thank you, @rainerhahnekamp. I’ll give it a try and get back to you.

**Comment by rainerhahnekamp** [comment]
Merged and available since 0.19.3
